### PR TITLE
Reboot reset

### PR DESCRIPTION
**Pull request by polachok**

### CODE REVIEW

File: libvirt.go [hunk 1]
```diff
@@ -260,6 +260,22 @@ const (
...
+	RebootAcpiPowerBtn RebootFlags = 1 << iota
+	// RebootGuestAgent - use guest agent
+	RebootGuestAgent
+	// RebootInitctl - use initctl
+	RebootInitctl
+	// RebootSignal - use signal
+	RebootSignal
+	// RebootParavirt - use paravirt guest control
+	RebootParavirt
+
```
Comment (benlemasurier) [inline libvirt.go]: remove this `\n`

File: libvirt.go [hunk 2]
```diff
@@ -260,6 +260,22 @@ const (
 	StoragePoolsFlagZFS
 )
 
+// RebootFlags specifies domain reboot methods
+type RebootFlags uint32
+const (
+	// RebootAcpiPowerBtn - send ACPI event
+	RebootAcpiPowerBtn RebootFlags = 1 << iota
```
Comment (benlemasurier) [inline libvirt.go]: `\n` after each `const` definition

File: libvirt.go [hunk 3]
```diff
@@ -918,6 +934,72 @@ func (l *Libvirt) Version() (string, error) {
 	return versionString, nil
 }
 
+// Reboot reboots the domain
```
Comment (benlemasurier) [inline libvirt.go]: this comment can be improved, it should also be a full sentence. Add a note about the behavior when flags is 0.

File: libvirt.go (hunk 2 above)
Comment (polachok) [inline libvirt.go]: fixed

**Comment by benlemasurier** [comment]
Looks great! We've got a conflict from #35, fix that up and I'll get this merged.

**Pull request merged**